utilities/ifix-migration-toolkit/src/main/resources/application.properties

```diff
@@ -55,7 +55,7 @@ ifix.migration.batch.size=200
 #DEPARTMENT ENTITY MIGRATION CONFIGURATION
 department.entity.migration.batch.size=200
 department.entity.migration.persister.topic=de-migration-topic
-persister.kafka.department.entity.create.topic=save-dept-entity-application
+persister.kafka.department.entity.create.topic=migrate-dept-entity-topic
```

Comment:
Why wouldn't a common migration topic work? What is the need to put records on a new topic. 
With this approach, we will need a separate persister config also.